docs/MACOS_INSTALL.md

```diff
@@ -0,0 +1,214 @@
+# macOS Installation Guide for RamaLama
+
+This guide covers the different ways to install RamaLama on macOS.
+
+## Method 1: Self-Contained Installer Package (Recommended)
+
+The easiest way to install RamaLama on macOS is using our self-contained `.pkg` installer. This method includes Python and all dependencies, so you don't need to install anything else.
```

Comment:
You'd still need to install podman. We can either add podman to the pkginstaller, or mention it here.